lua/refactoring/treesitter/treesitter.lua

```diff
@@ -122,6 +124,13 @@ function TreeSitter:get_local_defs(scope, region)
     return nodes
 end
 
+function TreeSitter:get_class_vars(scope, region)
+    -- TODO: add validate setting
+    local class_var_nodes = self:loop_thru_nodes(scope, self.class_vars)
+    print("vim.inspect(class_var_nodes):", vim.inspect(class_var_nodes))
```

Comment:
Was this an autogenerated print statement?

Reply:
Whoops, this should be removed. I'll open another to remove.